pkg/controller/catalogsource/catalogsource_controller.go

```diff
@@ -34,13 +33,12 @@ func newReconciler(mgr manager.Manager) reconcile.Reconciler {
 }
 
 func add(mgr manager.Manager, r reconcile.Reconciler) error {
-
 	c, err := controller.New("catalogsource-controller", mgr, controller.Options{Reconciler: r})
 	if err != nil {
 		return err
 	}
 
-	_, defaultCatalogsources := defaults.GetGlobalDefinitions()
+	defaultCatalogsources := defaults.GetGlobalCatalogSourceDefinitions()
```

Comment:
nit: Is the whole `GetGlobalCatalogSourceDefinitions`really necessary? In fact early on in the code base we used `Get` a lot and that just stuck around as convention. `GlobalDefinitions` seems sufficient here.